### ISSUE TITLE
Bugsnag Expo - TypeError (0, _expo.default) is not a function

### ISSUE THREAD

**Issue by danielneri** [bugsnag/bugsnag-js#856]
Upon installation of the notifier plugin for Expo @bugsnag/expo my app instantly throws the following exception on boot.

```
TypeError: (0, _expo.default) is not a function. (In '(0, _expo.default)()', '(0, _expo.default)' is an instance of Object)

<unknown>
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:125229:41
loadModuleImplementation
    require.js:322:6
<global>
    AppEntry.js:3
loadModuleImplementation
    require.js:322:6
guardedLoadModule
    require.js:201:45
global code
    AppEntry.bundle?platform=ios&dev=true&minify=false&hot=false:449972:4
```

Relevant sections of my package.json
```
    "@bugsnag/expo": "^7.0.1",
    "@react-native-community/masked-view": "0.1.6",
    "@react-native-community/netinfo": "^5.9.0",
    "@react-navigation/native": "^5.3.2",
    "@react-navigation/stack": "^5.3.6",
    "aws-amplify": "^3.0.11",
    "aws-amplify-react-native": "^4.1.2",
    "expo": "^37.0.0",
    "expo-auth-session": "~1.2.1",
    "expo-firebase-analytics": "~2.3.0",
    "expo-location": "~8.1.0",
    "expo-secure-store": "~8.1.0",
    "expo-web-browser": "~8.2.1",
    "firebase": "7.9.0",
    "moment": "^2.25.3",
    "prop-types": "^15.7.2",
    "react": "16.9.0",
    "react-dom": "16.9.0",
    "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.1.tar.gz",
    "react-native-datepicker": "^1.7.2",
    "react-native-gesture-handler": "~1.6.0",
    "react-native-maps": "0.26.1",
    "react-native-paper": "^3.10.1",
    "react-native-reanimated": "~1.7.0",
    "react-native-safe-area-context": "0.7.3",
    "react-native-screens": "~2.2.0",
    "react-native-svg": "11.0.1",
    "react-native-svg-transformer": "^0.14.3",
    "react-native-swiper": "^1.6.0",
    "react-native-web": "^0.11.7"
```

Including line 1 and 2 of my App.js

```
import bugsnag from '@bugsnag/expo';
const bugsnagClient = bugsnag();
```

**Comment by mattdyoung** [comment]
Hi @danielneri 

We made some changes to the interface for bugsnag-js v7, and it looks like you're using bugsnag-js v7.0.1 but still using the syntax from v6.

These lines here:
```
import bugsnag from '@bugsnag/expo'
const bugsnagClient = bugsnag()
```
should be changed to:
```
import Bugsnag from '@bugsnag/expo'
Bugsnag.start()
```

More detail is available in the documentation:
https://docs.bugsnag.com/platforms/react-native/expo/manual-setup/

and the upgrade guide:
https://github.com/bugsnag/bugsnag-js/blob/next/UPGRADING.md#6x-to-7x

**Comment by danielneri** [comment]
Thanks Matt - there is a bug in the npx script `bugsnag-expo-cli init`

The auto install lines are incorrect. 

Output from running the recommended command:
```
> Inserting API key into app.json
? The following Bugsnag initialization lines will be added to App.js. Is this ok?

  import bugsnag from '@bugsnag/expo';
  const bugsnagClient = bugsnag();
```

**Comment by mattdyoung** [comment]
That doesn't look right. We'll take a look. Thanks!

**Comment by mattdyoung** [comment]
Fixed in [v7.1.1](https://github.com/bugsnag/bugsnag-js/releases/tag/v7.1.1)
